docs/detections/alert-suppression.asciidoc

```diff
@@ -41,6 +41,8 @@ image::images/alert-suppression-options.png[Alert suppression options,400]
 
 TIP: Use the *Rule preview* before saving the rule to visualize how alert suppression will affect the alerts created, based on historical data.
 
+NOTE: If you're suppressing alerts based on a field that has multiple values, an alert grouping is created for each value. For example, if you select to suppress alerts by IP of `[a, b, c]`, alerts will be suppressed for those matching values `a`, `b`, and `c` separately.
```

Comment:
This might be better suited for step 1, since that's where users specify field names to suppress by. Also, suggested some minor edits but I'd to @joepeeples for the final feedback since he understands this feature better than I do. 

```suggestion
NOTE: If you specify a field with multiple values, an alert grouping is created for each value. For example, if you suppress alerts by IP of `[a, b, c]`, alerts will be suppressed separately for each matching value of `a`, `b`, and `c`.
```

Reply:
Super nit: might also be better to use something like `[127.0.0.1, 127.0.0.2, 127.0.0.3]` for the example. Sorry I didn't do that when creating the docs issue.

Reply:
Sounds good, thanks! Updated in [2e28a29](https://github.com/elastic/security-docs/pull/3879/commits/2e28a29d55373dac49f4817096f5e7cb03836380).